### PR TITLE
xgboost classifier example

### PR DESCRIPTION
**Pull request by oliviakim95**
@iandewancker @pfhayes @alexandraj777

### CODE REVIEW

File: xgboost-classifier/xgboost-softmax.R [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Use SigOpt to tune an XGBoost softmax classifier in R
+# Learn more about SigOpt's R Client:
+# https://sigopt.com/docs/overview/R
```
Comment (baieric) [inline xgboost-classifier/xgboost-softmax.R]: There was a change recently that broke https://sigopt.com/docs/overview/R
(not sure if we're planning on making it work again, cc @pfhayes)
You'll have to use lower-case r: https://sigopt.com/docs/overview/r

Comment (oliviakim95) [inline xgboost-classifier/xgboost-softmax.R]: oo okay thanks!

**Comment by pfhayes** [comment]
lgtm but @iandewancker should take a look

File: xgboost-classifier/xgboost-softmax.R [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  cv_accuracies = c()
+  for (k in 1:num_folds){
+    cv_split <- sample(2, nrow(iris), replace=TRUE, prob=c(1-1.0/num_folds, 1.0/num_folds))
+    X_train <- X[cv_split==1, ]
+    y_train <- y[cv_split==1]
+    X_valid <- X[cv_split==2, ]
+    y_valid <- y[cv_split==2]
+
+    boosted_clf <- xgb.train(
+      data=data,
```
Comment (iandewancker) [inline xgboost-classifier/xgboost-softmax.R]: ah I suppose this should be X_train, y_train and not the entire data matrix
